horizons/gui/modules/loadingscreen.py

```diff
@@ -51,7 +51,13 @@
 		"Travel beyond the horizon and discover unknown worlds!",
 		u"War… war never changes",
 		"Support Unknown Horizons with Cookies!",
-		"wow, looks nearly completed \n(Neomex)"
+		"wow, looks nearly completed \n(Neomex)",
+		"Anchor is missing ...",
+		"campfire is lighted",
```

Comment:
`campfire` starts with a lowercase character while all other quotes seem to start with a uppercase character.